proto-java-annotation/src/main/java/io/github/lwlee2608/proto/annotation/processor/ProtoProcessor.java

```diff
@@ -136,6 +137,10 @@ public boolean process(Set<? extends TypeElement> annotations, RoundEnvironment
                             protoType = getSimpleClass(javaType) + "Enum";
                             isEnum = true;
 
+                        } else if (javaType.startsWith("java.util.List")) {
+                            String subJavaType = javaType.substring(javaType.indexOf("<")+1, javaType.indexOf(">"));
+                            protoType = "repeated " + toProtoType(subJavaType);
```

Comment:
`protoType` should just be the raw data type.

Reply:
so we don't support null value in list<String> ?

Reply:
a `protoType` should be just `int32`. `string` and so on. `repeated int32` is not a protoType.
You should print `repeated` in `writeProtoFile()` function